projectFilesBackup1/.idea/workspace.xml

```diff
@@ -0,0 +1,4287 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
as you said - please remove this file and instead add an entry in .gitignore to ignore it.